models/release.go

```diff
@@ -244,6 +244,15 @@ func GetReleasesByRepoID(repoID int64, page, pageSize int) (rels []*Release, err
 	return rels, err
 }
 
+// GetReleaseCountByRepoID returns the count of releases of repository
+func GetReleaseCountByRepoID(repoID int64, isOwner bool) (total int64, err error) {
+	if isOwner {
+		return x.Where("repo_id = ?", repoID).Count(&Release{})
+	}
+
+	return x.Where("repo_id = ? AND is_prerelease = 0 AND is_draft = 0", repoID).Count(&Release{})
```

Comment:
I think you should use a Cond struct instead of a string, similar to [this](https://github.com/go-gitea/gitea/blob/d71fad2ab7afc5d1bfa17842b1a66c9d1d274167/models/issue_comment.go#L586-L598) and then you can pass it to `Where` method